src/components/form/login-form/index.tsx

```diff
@@ -0,0 +1,101 @@
+'use client'
+
+import { useState } from 'react'
+import { EyeIcon, EyeOffIcon } from 'lucide-react'
+
+import { Alert, AlertDescription } from '@/components/ui/alert'
+import { Button } from '@/components/ui/button'
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from '@/components/ui/card'
+import { Input } from '@/components/ui/input'
+import { Label } from '@/components/ui/label'
+
+export default function LoginForm() {
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+  const [showPassword, setShowPassword] = useState(false)
+  const [error, setError] = useState('')
+
+  const handleSubmit = (e: React.FormEvent) => {
+    e.preventDefault()
+    // Here you would typically handle the login logic
+    // For this example, we'll just show an error if the fields are empty
+    if (!email || !password) {
+      setError('Please fill in all fields')
+    } else {
+      setError('')
+      console.log('Login attempt with:', { email, password })
+      // Here you would typically make an API call to authenticate the user
+    }
+  }
```

Comment:
Don't we have sample for the form? This one is not following my convention I think